lib/nomade/http.rb

```diff
@@ -154,12 +154,34 @@ def plan_job(nomad_job)
       raise
     end
 
+    def dispatch_job(nomad_job, payload_data: nil, payload_metadata: nil)
+      if payload_metadata.class == Array && payload_metadata.empty?
```

Comment:
:thinking:

Reply:
Yes that is odd, nicely spotted @swistak35, we should just check for a Hash.

Reply:
I've changed this so it just checks for the class of payload_metadata.